uktrade/data-hub-frontend
# Feature/a11y integration

## Description of change

Utilise the existing implementation of Cypress accessibility to test all pages found in urls.js

## Test instructions

To run the tests pull this branch. Then start 4 terminal sessions to for the following instructions.

1. redis-server
2. cd test/sandbox && npm install && npx nodemon .
3. npm run develop
4. npm run test:a11y:watch

This will open Cypress in the browser. At the time of writing when the browser opens select **E2E Testing** and then start E2E testing in the browser of your choice.
Find all-accessibility.spec.js and run this test.

Alternatively for step 4. run **npm run test:a11y** to run the tests in the console.

Currently there should be 102 passing tests

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
